modules/server/server-rest/src/main/java/com/enonic/xp/impl/server/rest/SchedulerResource.java

```diff
@@ -0,0 +1,46 @@
+package com.enonic.xp.impl.server.rest;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import com.enonic.xp.impl.server.rest.model.ScheduledJobJson;
+import com.enonic.xp.jaxrs.JaxRsComponent;
+import com.enonic.xp.scheduler.SchedulerService;
+import com.enonic.xp.security.RoleKeys;
+
+@Path("/scheduler")
```

Comment:
maybe rename to `schedulers`, the GET method list will be available by default and that is right naming for REST

Reply:
but then we have to rename all exist resources